data/abilities.js

```diff
@@ -1373,7 +1373,7 @@ let BattleAbilities = {
 			if (!['surf', 'dive'].includes(move.id) || source.volatiles['dive'] || source.speciesid !== 'cramorant' || source.transformed) return;
 
 			// PLACEHOLDER Estimated 10% of the time you get pikachu
-			const forme = this.random(10) === 1 ? 'cramorantgorging' : 'cramorantgulping';
+			const forme = this.random(10) === 1 || source.hp <= Math.floor(source.maxhp / 2) ? 'cramorantgorging' : 'cramorantgulping';
```

Comment:
I don't understand. If it's 50%, why did you keep the 10% roll?

Reply:
it still has a random chance to change into Cramorant-Gorging above 50%

Reply:
Some tests that my brother did for me confirms its possible to get Gorging above 50%. Going to merge as is and get the exact percent later.

Reply:
After testing about 50 uses of surf cramorant with > 50% HP, Pikachu never appeared. So I'd say its supposed to be the fish when > 50%, and pikachu when < 50%.